src/Makefile

```diff
@@ -35,8 +35,6 @@ hoot: \
 	RPMS/x86_64/hootenanny-core-$(hootversion).el6.x86_64.rpm \
 
 hoot-deps: not-hoot
-	# Silly way to force a replacement of the gdal/geos RPMs
-	sudo yum -y remove gdal-devel gdal geos geos-devel
 	sudo yum -y install RPMS/x86_64/* RPMS/noarch/* libspatialite-devel postgis2_92
```

Comment:
Should we be able to remove the re-install of libspatialite-devel & postgis2_92?  I thought those were listed there because they were removed when their gdal & geos dependencies were removed.